pages/validator/operations/monitoring.md

```diff
@@ -0,0 +1,93 @@
+import Callout from 'nextra-theme-docs/callout'
+
+# Monitoring a Validator or Node
+
+There are a few different aspects to monitoring your validator. We need to monitor both the infrastructure and the axelard & supporting processes.
+
+## Infrastructure
+Infrastructure setups differ. This varies highly from one setup to another. Here are a few basic metrics to monitor.
+* CPU Usage
+* Memory Usage
+* Disk Usage (I/O)
+* Disk Space
+* Bandwidth
+
+You can run the node and quickly establish baselines for some of these like disk i/o, bandwidth etc.
+
+
+## Validator and Supporting Processes
+
+### Prometheus Metrics
+
+Axelar runs using Tendermint consensus. Several metrics are exposed via the prometheus metrics endpoint by Tendermint. You can find details on the metrics available through tendermint [here](https://docs.tendermint.com/v0.34/tendermint-core/metrics.html). All these metrics are prefixed with `tendermint` e.g `consensus_height` will be `tendermint_consensus_height`.
```

Comment:
Not sure if the prefix holds for the custom metrics we add.

Reply:
only talking about tendermint metrics mentioned in the docs. not ours.